components/post_view/commented_on/commented_on.jsx

```diff
@@ -32,24 +32,33 @@ export default class CommentedOn extends PureComponent {
     makeUsername = () => {
         const postProps = this.props.post.props;
         let username = this.props.displayName;
-        if (
-            this.props.enablePostUsernameOverride &&
-            postProps &&
-            postProps.from_webhook === 'true' &&
-            postProps.override_username
-        ) {
+        if (this.props.enablePostUsernameOverride && postProps && postProps.from_webhook === 'true' && postProps.override_username) {
             username = postProps.override_username;
         }
         return username;
     }
 
-    render() {
-        const {
-            post,
-        } = this.props;
+    makeCommentedOnMessage = () => {
+        const {post} = this.props;
+        let message = '';
+        if (post.message) {
+            message = Utils.replaceHtmlEntities(post.message);
+        } else if (post.file_ids && post.file_ids.length > 0) {
+            message = (
+                <CommentedOnFilesMessage parentPostId={post.id}/>
+            );
+        } else {
+            const attachment = post.props.attachments[0];
+            const webhookMessage = attachment.pretext || attachment.title || attachment.text || attachment.fallback;
```

Comment:
And can you add ` || '';` to the end of this in case someone creates an attachment without any of those fields?